### DISCUSSION TITLE
"Physical Review A" in bib loses the "A", whereas "Physical Review B" keeps its "B"

### DISCUSSION THREAD

**Discussion by Qottmann** [typst/typst#2135] [Q&A]
I am experiencing a weird behavior where `journal={Physical Review A}` gets displayed as "Physical Rev..", whereas `journal={Physical Review B}` gets correctly displayed as "Physical Rev. B". Is there any way to change this? Is this a bug or expected behavior?

I saw in the [guide for latex users](https://typst.app/docs/guides/guide-for-latex-users/) (very bottom) that apparently bbx style files are supported, however when I try to create such a file in the typst.app app I get an error saying unsupported file type.

Tags: biblography, physical review A B, bib, latex

**Comment by Qottmann** [comment]
A workaround is setting `journal={'Physical Rev. A'}`, which displays it exactly as it is, namely with the quotation marks. So it is not nice, but better than "Physical Rev.."

**Reply by xkevio** [reply]
I think the wording in the guide is unfortunate. It doesn't mean that Typst supports `.bbx` files but that LaTeX does with the aforementioned packages. Typst will use CSL at some point.
